action.yml

```diff
@@ -2,14 +2,6 @@ name: 'Add & Commit'
 description: 'Add & commit files from a path directly from GitHub Actions'
 
 inputs:
-  author_name:
```

Comment:
I still want an option to manually set name and email. You could just put a `fetch_author` option that, if checked, overrides any name or email and just uses the ones from the event file.

Reply:
why an option? how about keeping the input fields, but set them as not required and use The GitHub event if the input is not present.